cloud/scope/machine.go

```diff
@@ -26,13 +26,14 @@ type MachineScopeParams struct {
 }
 
 type MachineScope struct {
-	Client        K8sClient
-	PatchHelper   *patch.Helper
-	Cluster       *clusterv1.Cluster
-	Machine       *clusterv1.Machine
-	LinodeClient  LinodeClient
-	LinodeCluster *infrav1alpha2.LinodeCluster
-	LinodeMachine *infrav1alpha1.LinodeMachine
+	Client              K8sClient
```

Comment:
Given `K8sClient` is an interface that only embeds `client.Client`, could this field simply become 
```suggestion
	Client              client.Client
```
?

Reply:
Not sure about what the motive was behind whoever wrote this but I presume it was to avoid importing the client go library?

Reply:
That's fine, but if that is/was the reason, the client-go package is already being imported for the sake of the K8sClient interface.